src/rules/at-rule-name-newline-after/__tests__/index.js

```diff
@@ -0,0 +1,460 @@
+import { testRule } from "../../../testUtils"
+import rules from "../../../rules"
+import { ruleName, messages } from ".."
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: ["always"],
+  skipBasicChecks: true,
+
+  accept: [ {
+    code: "@charset\n\"UTF-8\";",
+  }, {
+    code: "@charset\r\n\"UTF-8\";",
+  }, {
+    code: "@import\n\"x.css\";",
+  }, {
+    code: "@import\r\n\"x.css\";",
+  }, {
+    code: "@import\n\"x.css\" screen and (orientation:landscape);",
+  }, {
+    code: "@import\r\n\"x.css\" screen and (orientation:landscape);",
+  }, {
+    code: "@import\nurl(\"x.css\");",
+  }, {
+    code: "@importn\r\nurl(\"x.css\");",
```

Comment:
I don't think every single test needs to be repeated with `\r\n`. If you just do a couple to demonstrate that behavior is the same, that will be fine and reduce clutter.